generators/server/index.js

```diff
@@ -118,8 +118,31 @@ module.exports = class extends ServerGenerator {
     }
 
     get install() {
+        const installPhaseFromJHipster = super._install();
+        const jhipsterInstallNodeSteps = {
+            jhipsterNodeServerInstall() {
+                if (this.skipServer) return;
+                const logMsg = `To install your server dependencies manually, run: cd server && ${chalk.yellow.bold(
+                    `${this.clientPackageManager} install`
+                )}`;
+
+                if (this.options.skipInstall) {
```

Comment:
@mraible , do you know a way to exclude [the lines about npm install]() from coverage?

Reply:
I do not know, but maybe @mshima does?

Reply:
Sorry, never used.
Looks like `/*istanbul ignore next*/` should be the answer.
https://stackoverflow.com/questions/30327830/excluding-files-from-coverage-when-using-mocha-and-istanbul

Reply:
I saw that issue on stackoverflow. I try with it, also because there is indicated in the [istanbul doc](https://github.com/istanbuljs/nyc#parsing-hints-ignoring-lines)

Reply:
It works, thanks!